page_wrapper.go

```diff
@@ -103,6 +103,10 @@ func (p *pageWrapper) Sleep(time float64) {
 	p.Page.WaitForTimeout(time)
 }
 
+func (p *pageWrapper) WaitForTimeout(time float64) {
+	p.Page.WaitForTimeout(time)
```

Comment:
Does this return error at all? Should we 
```suggestion
	if err := p.Page.WaitForTimeout(time); err != nil {
		log.Fatalf("error with waiting for timeout: %v", err)
	}
```

Reply:
Page.WaitForTimeout does not return error, in-fact it has void return type. So I think we are good here.

Reply:
Ah odd, this is cool then